configs/native.yaml

```diff
@@ -1,16 +1,15 @@
 plugins:
   - path: ground_truth_slam/
   - path: offline_imu_cam/
-  - path: open_vins/
+  - path: git+https://github.com/charmoniumQ/open_vins.git
   - path: pose_prediction/
   - path: gldemo/
   - path: timewarp_gl/
   - path: debugview/
   - path: audio_pipeline/
   # - path: hologram/
   # - path: zed/
-#data: zip+http://robotics.ethz.ch/~asl-datasets/ijrr_euroc_mav_dataset/vicon_room1/V1_02_medium/V1_02_medium.zip?path=mav0
-data: ../../ILLIXR/data1/
+data: file+zip+http://robotics.ethz.ch/~asl-datasets/ijrr_euroc_mav_dataset/vicon_room1/V1_02_medium/V1_02_medium.zip#path=mav0
```

Comment:
We should either update our docs explaining this url setup or have the yaml setup in a way that the user doesn't need to supply the url int this setup and can instead tell us in other ways what type it is (zip, tar, git, etc.) or we infer it by parsing the actual url.

Reply:
I added this to the documentation; the `zip` scheme can only operate with zip archives (not tar or anything else). This is a current deficiency that we can fix later.